src/types.ts

```diff
@@ -0,0 +1,57 @@
+
+import React, { Dispatch } from "react"
+
+export type sectionData = {
+    name: string,
+    description: string,
+    numRevisions: number
+}
+
+export type sectionDataProps = {
+    sectionData: sectionData
+}
+
+
+
+export type chordProgression = string[]
+
+export type chordProgressionProps = {
```

Comment:
Props are typically defined right above the component that accepts those props, and are usually not exported either. Any parent component rendering this component infers the prop types automatically. Even though the type would not be exported, it implicitly is when calling/rendering the component since it accepts that as an argument